docs/_layouts/default.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
It's ok lowercase (= never have to worry about lower or uppercase so).

html5 boilerplate is ok: https://github.com/h5bp/html5-boilerplate/blob/master/dist/index.html#L1